models/lung_nodule_ct_detection/docs/README.md

```diff
@@ -33,7 +33,7 @@ Alternatively, we provide [resampled nifti images](https://drive.google.com/driv
 ## Training configuration
 The training was performed with the following:
 
-- GPU: at least 16GB GPU memory
+- GPU: at least 16GB GPU memory, requires 24G when exporting TRT model
```

Comment:
We hit an `OOM` error with L40G 24GB right when exporting TRT, didn't we?

Reply:
I changed to 32G. I thought it means it requires 24G. So it requires 32G for TRT? What is the actual requirement?

Reply:
cc @binliunls

Reply:
Hi @mingxin-zheng ,
    I ran several tests on A100/V100. The memories are different on different GPUs. The max one is a little bit greater than 24G. So I would suggest to use GPUs, whose memory is greater than (not equal to) 24G. 32G would be a nice choice.

Thanks,
Bin

Reply:
Thanks!